/**:
  ros__parameters:
    velocity: 1.0 # m/s
    travel_distance: 7.0 # m
    control_rate_hz: 10.0 # Hz
    yaw: 0.0 # radians
    yaw_rate: 0.25 # radians/s
    takeoff_height: 1.0 # m
    takeoff_wait_time: 10.0 # s
